### PR TITLE
Switch to latest

### PR DESCRIPTION
**Pull request by tomap**
remove nodejs 6 support

### CODE REVIEW

**Comment by tomap** [comment]
note that I did not managed to make this work.
There is still a failing test and I don't know how to fix it

related to https://github.com/hexojs/hexo/issues/3508

**Comment by weyusi** [comment]
https://github.com/hexojs/hexo-server/blob/518fe819aee389e9ff7a130c061123124c8ad903/test/index.js#L153

Change it to
```js
    .expect(200, {})
```

**Comment by tomap** [comment]
@weyusi thank you. it worked!

File: package.json [hunk 1]
```diff
@@ -24,30 +24,30 @@
...
+    "istanbul": "^0.4.5",
+    "mocha": "^6.1.4",
+    "sinon": "^7.3.2",
+    "supertest": "^4.0.2",
     "supertest-promised": "^1.0.0",
     "uuid": "^3.3.2"
   },
   "engines": {
-    "node": ">=6.9.0"
+    "node": ">=8"
```
Comment (YoshinoriN) [inline package.json]: IMO `>=8.6.0` is better. It's the same with https://github.com/hexojs/hexo/pull/3598

Comment (tomap) [reply package.json]: Done

**Comment by weyusi** [comment]
for appveyor, I suggest [8, 10, 12].
Currently LTS and Stable are 10. I'm not so keen on LTS because it usually refers to more than one versions.

**Pull request merged**